src/map/script.c

```diff
@@ -23384,6 +23384,33 @@ BUILDIN(activatepset);
 BUILDIN(deactivatepset);
 BUILDIN(deletepset);
 
+/**
+ * opendressroom(true/false);
+ */
+BUILDIN(opendressroom)
+{
+#if PACKETVER >= 20150513
+	struct map_session_data *sd = script->rid2sd(st);
+	
+	if (sd == NULL)
+		return false;
+
+	if (!script_hasdata(st, 2)) {
+		clif_dressroom_open(sd, 1);
```

Comment:
Please use interfaces and not direct function calls
clif_dressroom_open -> clif->dressroom_open